### ISSUE TITLE
Feature Request: Allow custom post process effects on ViroARScene

### ISSUE THREAD

**Issue by dam00n** [viromedia/viro#259]
Please allow the creation of custom post process effect of ViroARScene and/or support an animatable scalar value of the post process effect strength to allow for fade/ease animation when changing a post process effect.

**Comment by jaxzin** [comment]
@radvani Closed because the feature was implemented? Or closed because you're pruning the backlog?

**Comment by radvani** [comment]
This was pruning. However we're also gauging interest in various features -- is this something you're interested in?

**Comment by jaxzin** [comment]
Yes, interested in this. As the built-in post process options are fun but limited. We were building a prototype with Viro and the lack of this feature pushed us in a different direction. Being able to write a custom image filter would allow us to continue to use ViroARScene.

**Comment by Jip-Hop** [comment]
I'm interested in this too!

**Comment by marcoburrometo** [comment]
I need this too :)
